File: src/Enterprise/C#/patterns/EnterpriseFacade.cs
namespace Genkidama.PatternExamples;
public static class EnterpriseFacadeExample { public static bool Run(){string Crm(int i)=>$"crm:create:{i}";string Billing(int i)=>$"billing:open:{i}";return $"{Crm(77)}>{Billing(77)}"=="crm:create:77>billing:open:77";} }

File: src/Functional/Scala/patterns/EnterpriseFacade.scala
object EnterpriseFacadeExample { def run:Boolean={def crm(i:Int)=s"crm:create:$i";def billing(i:Int)=s"billing:open:$i";s"${crm(77)}>${billing(77)}"=="crm:create:77>billing:open:77"} }
